src/loqedAPI/loqed.py

```diff
@@ -224,6 +225,7 @@ async def receiveWebhook(self, body, hash: str, timestamp: str):
                 if "night_lock" in self.last_event: self.bolt_state="locking"
                 if "open" in self.last_event: self.bolt_state="opening"
                 if "latch" in self.last_event: self.bolt_state="unlocking"
+                self.go_to_state = str.replace(self.last_event,"go_to_state_","")
```

Comment:
I think this logic won't work universally. 
For example if `last_event` is `MOTOR_STALL` it would result in `go_to_state` = `MOTOR_STALL`.
I would check if `last_event` is starting with `go_to_state_` first.